src/thunderbots/software/util/math_functions.cpp

```diff
@@ -2,6 +2,8 @@
 // Created by roark on 28/03/19.
 //
 
+#include "util/math_functions.h"
+
 #include <algorithm>
 
 namespace Util
```

Comment:
Could we leave this outside of the `Util` namespace for now? I'd like to see this functions in the same namespace as the geom stuff, which is just no namespace for now (we're for sure going to re-evaluate all the namespaces after competition).

Reply:
moved